test/components/evaluators/test_context_relevance_evaluator.py

```diff
@@ -0,0 +1,118 @@
+from typing import List
```

Comment:
Let's also add an integration test that calls the OpenAI API.

Reply:
done 👍